charts/ci-operator/templates/steps/standard.yaml

```diff
@@ -86,6 +86,460 @@ step:
   script: |
     gitleaks -v --pretty --repo-path $(resources.inputs.source.path) --commit-from=$$(params.commitFrom) --commit-to=$(params.commitBefore) --branch=$(params.branch) --report gitleaks-report.json
 ---
+apiVersion: ci.w6d.io/v1alpha1
+kind: Step
+metadata:
+  name: static-test-gitleakspathscan
+  labels:
+  {{- include "ci-operator.labels" . | nindent 4 }}
+  annotations:
+    ci.w6d.io/kind: generic
+    ci.w6d.io/task: static-test
+    ci.w6d.io/order: "0"
+params:
+  - name: CLIENT_HTTP_REPOSITORY_URL
+    description: defines the http(s) repository
+    type: string
+step:
+  name: static-test-gitleakspathscan
+  image: "zricethezav/gitleaks:latest"
+  script: |
+    mkdir -p $(workspaces.source.path)/tests/report-gitleaks
+    cd $(workspaces.source.path)/tests/
```

Comment:
the report should not be in the source but $(workspaces.artifacts.path)

Reply:
done